packages/component-library/src/Header/index.module.scss

```diff
@@ -219,4 +219,87 @@
       }
     }
   }
+
+  .communityLinkList {
+    display: flex;
+    flex-flow: column nowrap;
+    gap: theme.spacing(4);
+
+    .title {
+      @include theme.text("lg", "medium");
+
+      color: theme.color("heading");
+    }
+
+    .communityGrid {
+      display: grid;
+      grid-template-columns: repeat(auto-fit, minmax(200px, 1fr));
+      gap: theme.spacing(3);
+      width: 100%;
+
+      @include theme.breakpoint("sm") {
+        grid-template-columns: repeat(auto-fit, minmax(160px, 1fr));
+        gap: theme.spacing(2);
+      }
+    }
+
+    .communityCard {
+      aspect-ratio: 1;
+      min-height: 160px;
+      padding: theme.spacing(6);
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      text-align: center;
+      transition:
+        transform 0.2s ease,
+        box-shadow 0.2s ease;
+
+      &:hover {
+        transform: translateY(-2px);
+        box-shadow: 0 8px 25px rgb(0 0 0 / 15%);
+      }
+
+      @include theme.breakpoint("sm") {
+        min-height: 140px;
+        padding: theme.spacing(4);
+      }
+    }
+
+    .communityCardContent {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      gap: theme.spacing(3);
+      width: 100%;
+      height: 100%;
+      justify-content: center;
+    }
+
+    .communityIcon {
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      width: theme.spacing(12);
+      height: theme.spacing(12);
+      color: theme.color("states", "data", "normal");
+
+      svg {
+        width: 100%;
+        height: 100%;
+      }
```

Comment:
This styling should be unneeded, since you're using phosphor icons the default width and height are set to `1em`.  This means that the right way to style them is to set `font-size: theme.spacing(12)` on `.communityIcon` instead of setting `width` and `height` and having the styling here.